src/background/user-storage.ts

```diff
@@ -45,6 +45,11 @@ export default class UserStorage {
         this.settings = null;
     }
 
+    getDefaultSettings() {
+        return this.defaultSettings;
```

Comment:
Let's move an object, assigned to `this.defaultSettings`, into `src/default-settings.ts` file. It will look like `export const DEFAULT_SETTINGS: UserSettings = {...};`. Then it can be reused.